02LessonTwo/homework/index.html

```diff
@@ -1,10 +1,39 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title></title>
-  <link rel="stylesheet" href="css/style.css">
+  <title>Khadijah's Bio</title>
 </head>
 <body>
-  <h1>Indian Restaurant</h1>
+  <h1><u>Khadijah's Bio</u></h1>
+  <a ref="">I own a company called Best Lead Generation that
+    specializes in providing <div> business-to-business lead generation
+    services to companies in order to help <div>grow their business.</a>
+
+    <h3><u>Things I like:</u><h3>
+    <ul>
+    <li> Reading
+    <li> Hiking
+    <li> Baking
+    <li> Cooking
+    <li> Coding
+    </ul>
+
+<h3><u>Things I don't like:</u><h3>
+    <ol>
+    <li> Bananas
+    <li> Bad traffic
+    <li> Bad weather
+    </ol>
+
+<h3><u>TV shows I like:</u><h3>
+    <ul>
+    <li> The Walking Dead
+    <li> Scandal
+    <li> Underground
+      </ul>
+
+<h3><u>Favorite Websites:</u><h3>
+  <ul>
+    <li>www.bestleadgen.com</li>
```

Comment:
Don't forget to close your `<ul>` tag after the last list item. You didn't do it here but the browser was forgiving and still rendered your content as intended.